cub/test/test_grid_barrier.cu

```diff
@@ -47,7 +47,8 @@ using namespace cub;
 /**
  * Kernel that iterates through the specified number of software global barriers
  */
-__global__ void Kernel(GridBarrier global_barrier, int iterations)
+_CCCL_SUPPRESS_DEPRECATED_PUSH
+__global__ void Kernel(GridBarrier global_barrier, int iterations) _CCCL_SUPPRESS_DEPRECATED_POP
```

Comment:
The `_CCCL_SUPPRESS_DEPRECATED_POP` is moved there by clang-format. I had it on a separate line below.

Reply:
I tried adding the push/pop macros to `StatementMacros`, but it didn't help :/ So I enforce a line break by a trailing line comment.